app/src/test/java/edu/uc/app/parkfinder/ExampleUnitTest.kt

```diff
@@ -1,17 +1,22 @@
 package edu.uc.app.parkfinder
-
+import edu.uc.app.parkfinder.dto.Park
 import org.junit.Test
-
 import org.junit.Assert.*
 
+
 /**
  * Example local unit test, which will execute on the development machine (host).
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class ExampleUnitTest {
+
     @Test
-    fun addition_isCorrect() {
-        assertEquals(4, 2 + 2)
+    fun parkDTO_maintainsPark(){
```

Comment:
Got rid of old unit test and started new proper unit tests for new Parks JSON endpoint provided.

Reply:
Very good.